src/components/App.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import "./App.scss";
+import Routes from "../routes";
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <h1 className="app-name">Author's Haven</h1>
+        <h2 className="team-name">LIT Kigali</h2>
+        <Routes />
+      </div>
+    );
+  }
+}
+// const App = () => (
```

Comment:
why did we push our work with commented lines?

Reply:
App always has to be a stateful class... But I guess you learned that.

Reply:
Ok I am to fix that.